man/mariadb-admin.1

```diff
@@ -800,6 +800,22 @@ Display help and exit\&.
 .sp -1
 .IP \(bu 2.3
 .\}
+.\" mysql: bind-address option
+.\" bind-address option: mysql
+\fB\-\-bind\-address=\fR\fB\fIip_address\fR\fR
+.sp
+On a computer having multiple network interfaces, use this option to select
+which interface to use for connecting to the MySQL server\&.
```

Comment:
can we call it "the MariaDB server". There are a few lingering instances of MySQL server in the man pages but we'll get to those.

Reply:
Sure. I was also wondering if I could use MariaDB here since I didn't know the intention. Fixed.